frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -353,9 +403,18 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     }
   };
 
+  const closeOverlay = () => {
+    const params = new URLSearchParams(window.location.search);
+    params.delete('details-item');
+    setURLParams(params);
+    setDetailsItem(null);
+    setShowDetails(false);
+  };
+
   const openOverlay = (item: OperatorHubItem) => {
     if (!ignoreOperatorWarning && item.providerType === COMMUNITY_PROVIDER_TYPE) {
       communityOperatorWarningModal({
+        close: closeOverlay,
```

Comment:
Clicking 'Continue' button in `Show Community Operator` modal will close the modal instead of redirecting to operator installation page, this should be fixed
<img width="679" alt="Screen Shot 2020-12-16 at 4 55 14 PM" src="https://user-images.githubusercontent.com/12692381/102326695-adac8480-3fbf-11eb-9fb3-a935167a8a95.png">

Reply:
this is updated